dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExporterTest.java

```diff
@@ -3,14 +3,13 @@
 import edu.harvard.iq.dataverse.export.OpenAireExporter;
 import edu.harvard.iq.dataverse.settings.SettingsServiceBean;
 import edu.harvard.iq.dataverse.util.json.JsonPrinter;
-import org.junit.jupiter.api.Test;
```

Comment:
We should use single imports, it's easier to see in review later which class is used, you can adjust it in IDE to do it automatically

Reply:
Actually IDE prompts me to turn off:
![image](https://user-images.githubusercontent.com/51329176/132660932-493c2f23-6e32-4c60-89fe-1b02197c8e57.png)

Also it says that some standards prohibits using it so I'm not convinced that we should.

Reply:
I can't see the prompt :P, well I think we always used single imports in this project but it's not very important so your call.

Reply:
Since optimizing import does nothing to change it (with count for imports set to 999) I'll leave it as is. IDE does not want to change it for some reason.

Reply:
I think if you remove those imports and reimports then it will do it properly.

Reply:
Nope :P

Reply:
![image](https://user-images.githubusercontent.com/22947763/132662653-57c5612a-430c-4738-a1d1-f948dc107f84.png)
That's my settings :p